### PR TITLE
Port account_invoice_shipping_address to 8.0

### PR DESCRIPTION
**Pull request by anthony-muschang**
Migration of the module account_invoice_shipping_address to 8.0.

The implementation to generate invoice from a stock.picking have changed in 8.0, a little adaptation have been necessary. I add a test to valid those modifications.

### CODE REVIEW

**Comment by pedrobaeza** [comment]
:+1: 

Thanks

**Comment by adrienpeiffer** [comment]
:+1: (Code review)

File: account_invoice_shipping_address/README.rst [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+This module adds a shipping address field to the invoice, also it fills it in sale and picking invoicing
+
+Credits
+=======
+
+Contributors
+------------
+
+* Leonardo Pistone <leonardo.pistone@domsense.com>
```
Comment (eLBati) [inline account_invoice_shipping_address/README.rst]: This address does not exist anymore. @lepistone do you want to change it?

Comment (anthony-muschang) [inline account_invoice_shipping_address/README.rst]: @eLBati @lepistone The email address has been updated.

**Comment by Olivier-LAURENT** [comment]
:+1:

**Pull request merged**